### ISSUE TITLE
remove context button lacks aria label

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#238182]
Discovered with @jooyoungseo

It is confusing because:
1) Lacks a label
2) When `shift+tab` vs `tab` are used, a user doesn't know to which file the button belongs.

I suggest we add the filename to the aria label for the remove context button... e.g. "Remove context file/path"

https://github.com/user-attachments/assets/403220a5-0721-4833-9a40-396d0d88d76f
